teachertool/src/components/HeaderBar.tsx

```diff
@@ -29,28 +30,28 @@ export const HeaderBar: React.FC<HeaderBarProps> = () => {
     const getTargetLogo = () => {
         return (
             <div
+                className={css["brand"]}
                 aria-label={lf("{0} Logo", appTheme.boardName)}
                 role="menuitem"
-                className={`ui item logo brand mobile hide`}
                 onClick={brandIconClick}
             >
                 {appTheme.useTextLogo ? (
                     [
-                        <span className="name" key="org-name">
+                        <span className={css["name"]} key="org-name">
                             {appTheme.organizationText}
                         </span>,
-                        <span className="name-short" key="org-name-short">
+                        <span className={css["name-short"]} key="org-name-short">
                             {appTheme.organizationShortText || appTheme.organizationText}
                         </span>,
                     ]
                 ) : appTheme.logo || appTheme.portraitLogo ? (
                     <img
-                        className={`ui ${appTheme.logoWide ? "small" : ""} logo`}
+                        className={css["logo"]}
```

Comment:
It may be necessary to support the a special layout for `logoWide` in the future. But if so, we should implement it using responsive classes defined in react-common, and avoid use of classes defined in core webapp css. They're not well-suited to use outside the webapp.